public/styles.css

```diff
@@ -758,7 +758,10 @@ td.right, th.right {
 td.row-id {
     font-weight: bold;
 }
-
+.row {
```

Comment:
We already have this style in binary-styles that we use.
Also, almost always use rems for this in this case about 0.5rems